backend/offchain/types/__init__.py

```diff
@@ -355,9 +363,161 @@ def new_payment_info_object(
+                sender_line1,
+                sender_line2,
+                sender_name,
+                sender_sure_name,
+                sender_national_id_type,
+                sender_national_id_value,
+                sender_postal_code,
+                sender_state,
+                vasp_address,
+            ),
+        ),
+    )
+
+
+def new_init_charge_command(
+    reference_id,
+    vasp_address,
+    sender_name,
+    sender_sure_name,
+    sender_city,
+    sender_country,
+    sender_line1,
+    sender_line2,
+    sender_postal_code,
+    sender_state,
+    sender_national_id_value,
+    sender_national_id_type,
+):
+    return CommandRequestObject(
+        cid=reference_id or str(uuid.uuid4()),
+        command_type=CommandType.InitChargePayment,
+        command=InitChargePayment(
+            reference_id=reference_id,
+            sender=new_payment_sender_object(
+                sender_city,
+                sender_country,
+                sender_line1,
+                sender_line2,
+                sender_name,
+                sender_sure_name,
+                sender_national_id_type,
+                sender_national_id_value,
+                sender_postal_code,
+                sender_state,
+                vasp_address,
+            ),
+        ),
+    )
+
+
+def new_payment_sender_object(
+    city,
+    country,
+    line1,
+    line2,
+    my_name,
+    my_sure_name,
+    national_id_type,
+    national_id_value,
+    postal_code,
+    state,
+    vasp_address,
+):
+    return PaymentSenderObject(
+        account_address=vasp_address,
+        payer_data=new_payer_data_object(
+            city,
+            country,
+            line1,
+            line2,
+            my_name,
+            my_sure_name,
+            national_id_type,
+            national_id_value,
+            postal_code,
+            state,
+        ),
+    )
+
+
+def new_payer_data_object(
+    city,
+    country,
+    line1,
+    line2,
+    my_name,
+    my_sure_name,
+    national_id_type,
+    national_id_value,
+    postal_code,
+    state,
+):
+    return PayerDataObject(
+        given_name=my_name,
+        surname=my_sure_name,
+        address=new_address_object(
+            city=city,
+            country=country,
+            line1=line1,
+            line2=line2,
+            postal_code=postal_code,
+            state=state,
+        ),
+        national_id=new_national_id_object(
+            country, national_id_type, national_id_value
+        ),
+    )
+
+
+def new_national_id_object(country, national_id_type, national_id_value):
+    return NationalIdObject(
+        id_value=national_id_value,
+        country=country,
+        type=national_id_type,
+    )
+
+
+def new_address_object(city, country, line1, line2, postal_code, state):
+    return AddressObject(
+        city=city,
+        country=country,
+        line1=line1,
+        line2=line2,
+        postal_code=postal_code,
+        state=state,
+    )
```

Comment:
I'd keep all these new_* functions inside the class the instantiate – as static methods. Will it break something if they are moved there?

Reply:
but the new_address_object is not relevant for specific object, we use it for PaymentCommand and GetPaymentInfo, so I assume this methods may be reuse to future objects as well

Reply:
I meant that the user, instead of writing 
```
addr = new_address_object(...)
```
would write something like this:
```
addr = AddressObject.create(...)
```